compiler/luci/pass/src/RemoveUnnecessaryTransposeNetPass.cpp

```diff
@@ -19,11 +19,62 @@
 #include <luci/IR/CircleNodes.h>
 #include <luci/Profile/CircleNodeOrigin.h>
 
+#include <limits>
 #include <vector>
 
 namespace
 {
 
+#define CHECK_OR_FALSE(condition) \
+  if (not(condition))             \
+    return false;
+
+bool extract_shape(const luci::CircleNode *node, std::vector<int32_t> &shape)
+{
+  uint32_t max_i32 = static_cast<uint32_t>(std::numeric_limits<int32_t>::max());
+
+  auto rank = node->rank();
+  for (auto i = 0u; i < rank; ++i)
+  {
+    uint32_t v = node->dim(i).value();
```

Comment:
Is it okay to not consider the 64-bit range?

```suggestion
    uint64_t v = node->dim(i).value();
```

Reply:
Thank you for the review! 

https://github.com/Samsung/ONE/blob/a661f4b2ced72b2b3c1a723b2603344b25c9a2f5/compiler/loco/include/loco/IR/Dimension.h#L51  

`loco::Dimension` only holds `uint32_t` values. So we don't need to handle `int64_t` for this function.